app/views/rubygems/_gem_members.html.erb

```diff
@@ -1,28 +1,22 @@
 <div class="gem__members">
   <% if rubygem.owners.present? %>
     <h3 class="t-list__heading"><%= t '.owners_header' %>:</h3>
-    <div class="gem__users">
-      <%= links_to_owners(rubygem) %>
-    </div>
+    <ul class="tw-my-3">
+      <% show_mfa_status = policy(rubygem).show_owner_mfa_status? %>
+      <% rubygem.owners.sort_by(&:id).each do |owner| %>
+        <li>
+          <%= link_to_owner(owner, show_mfa_status:) %>
+        </li>
+      <% end %>
+    </ul>
 
-    <% if  rubygem.owned_by?(current_user) %>
+    <% if rubygem.owned_by?(current_user) %>
```

Comment:
this should also use the `show_owner_mfa_status?` check imo

Reply:
agreed.